packages/client/components/NewMeetingCarousel.tsx

```diff
@@ -86,24 +87,49 @@ interface Props {
   idx: number
   setIdx: (idx: number) => void
   meetingOrder: MeetingTypeEnum[]
+  onStartMeetingClick: () => void
 }
 
 const NewMeetingCarousel = (props: Props) => {
-  const {idx, setIdx, meetingOrder} = props
+  const {idx, setIdx, meetingOrder, onStartMeetingClick} = props
 
   // TODO: remove when standups feature flag removed
   const moreThanThreeSlides = meetingOrder.length > 3
 
+  useHotkey('left', () => {
```

Comment:
`swiper` package doesn't recognise left or right key down events, I _think_ because their keyboard functionality lets you swipe using the up & down arrow key. I changed the direction to `horizontal`, but no dice. Went with `useHotKey` to build something quickly

Reply:
In current implementation of carousel selected slide in swiper is not synchronized with the selected slide in our modal. 

It is kind of different concepts:

For example when we drag a view, our selected slide don't change. However, in swiper selected slide changes.  Selected slide for us is a slide that we clicked on and highlighted. Selected slide for swiper is the first slide in the view.

So I think your implementation is good.

And probably we can replace swiper package later with our own implementation.

Reply:
Ah great point! I didn't have standups enabled locally, so there were only 3 meeting types.

It looks like there's a bug, probably with the package, that doesn't update the active CSS correctly. For example, when I click "Async Standup", the "Retrospective" still has the `swiper-slide-active` class:

![active-idx](https://user-images.githubusercontent.com/39854876/182356279-9587ed52-515a-4012-b614-b4eb294f8a1a.png)

I spent some time looking into the package but didn't find a solution, so for now if `idx === 0` and the press left, they'll stay on idx 0. I think you're right that we should probably replace the swiper package with our own implementation.